tests/functional/test_install.py

```diff
@@ -1253,13 +1263,14 @@ def test_install_editable_with_prefix_setup_py(script: PipTestEnvironment) -> No
     _test_install_editable_with_prefix(script, {"setup.py": setup_py})
 
 
+@pytest.mark.network
 def test_install_editable_with_prefix_setup_cfg(script: PipTestEnvironment) -> None:
```

Comment:
Hmm, why is this one incompatible?

Reply:
@uranusjr that's because `_test_install_editable_with_prefix` tests that a `.egg-link` file has been created.

Reply:
Should we either fix the check, or add an additional test for modern editable?

Reply:
Maybe. I'll likely forget during my holidays, though :)